validator/example/cmd_test.go

```diff
@@ -11,15 +11,24 @@ func Test_ExecuteCommand(t *testing.T) {
 
 	// Testing cobra commands with default recommended config
 	// default config can also be overrided
-	var vali rules.RuleConfig
-	validationErr := vali.ExecuteRules(cmd)
+	ruleCfg := rules.ValidatorConfig{
+		ValidatorOptions: rules.ValidatorOptions{Verbose: false},
```

Comment:
Do we need to specify value that is used by default?

```suggestion
```

Reply:
@wtrocki review required

Reply:
Of the whole PR? or just this?

Reply:
whole

Reply:
now merged

Reply:
🙈